javascript/nanoexpress-pro/config.yaml

```diff
@@ -0,0 +1,5 @@
+framework:
+  github: dalisoft/nanoexpress#pro
+  website: nanoexpress.js.org
```

Comment:
could you use only one markup, rather `website`

Reply:
Ok, i will fix it